src/serlo-editor-repo/math/visual-editor.tsx

```diff
@@ -95,6 +95,12 @@ export function VisualEditor(props: VisualEditorProps) {
       onChange={(ref) => {
         props.onChange(ref.latex())
       }}
+      onCopy={(event: React.ClipboardEvent) => {
+        event.stopPropagation()
+      }}
+      onCut={(event: React.ClipboardEvent) => {
+        event.stopPropagation()
+      }}
```

Comment:
Do you see a need in the near future to let this event bubble up further to the parent event handlers? Maybe when we select and copy/cut some text and inline math elements in one go.

Reply:
I don't really see that need.

In the case you mentioned, the math editor is not focused, so its copy and cut handlers won't be triggered anyway, the whole math element will be copied instead.

Reply:
I guess the math editor will be focused when the selection ends within a math editor. But I think this use case is not happening very often.

Reply:
The selection currently works so that you cannot select a part of the math editor, when already selecting outside the math editor. Only the whole math element can be selected in that case.